src/app/src/test/kotlin/org/imperial/mrc/hint/integration/CleanDatabaseTests.kt

```diff
@@ -27,6 +28,11 @@ abstract class CleanDatabaseTests
 
         val tableFields = Tables::class.java.fields
 
+        // this table has foreign keys
+        // so has to be deleted first
+        dsl.deleteFrom(SESSION_FILE)
```

Comment:
It looks like Jooq should support truncate with cascade, which I think should avoid needing to sort out foreign keys first. 
https://www.jooq.org/javadoc/3.7.1/org/jooq/TruncateCascadeStep.html

Reply:
ah brill, thanks